utils/emoticons.tsx

```diff
@@ -26,12 +26,13 @@ export const emoticonPatterns: { [key: string]: RegExp } = {
 export const EMOJI_PATTERN = /(:([a-zA-Z0-9_+-]+):)/g;
 
 export function matchEmoticons(text: string): RegExpMatchArray | null {
-    let emojis = text.match(EMOJI_PATTERN);
+    const markdownCleanedText = text.replace(/`(.*?)`/g, '');
```

Comment:
I can move this part to utils. Since as it is only used here maybe it makes sense to keep it here. Please let me know what do you prefer.